addons/behaviour_toolkit/finite_state_machine/fsm_state_integrated_bt.gd

```diff
@@ -37,3 +75,27 @@ func _get_behaviour_tree() -> BTRoot:
 			return child
 
 	return null
+
+
+func _disable_autostart() -> void:
+	if Engine.is_editor_hint():
+		return
+	
+	# To prevent wrong configuration from the user side disable autostart.
+	# States BTree should be run only when state is active.
+	# TODO make editor inspector disable autostart when BTree is a child
+	# of FSMStateIntegratedBT?
+	if get_child_count() == 0:
+		return
+	
+	for child in get_children():
+		if child is BTRoot:
+			child.autostart = false
+
```

Comment:
Is this really necessary? Autostart is set to `false` on default. Maybe using a configuration warning is more appropriate here. 

> make editor inspector disable autostart when BTree is a child

This is also a good idea. The inspector plugin can hide the autostart option and set it to `false` automatically.